app/src/lib/git/commit.ts

```diff
@@ -94,6 +120,55 @@ export async function createMergeCommit(
   }
 }
 
+async function stageManualConflictResolution(
+  repository: Repository,
+  file: WorkingDirectoryFileChange,
+  manualResolution: ManualConflictResolution
+): Promise<boolean> {
```

Comment:
What does the return type imply here? It's not obvious what `true` or `false` means. Looking above at https://github.com/desktop/desktop/pull/6606/files#diff-7f18b49a6d37b89590132862b18c161aR72 also provides no hints since we don't capture the result of this function call. I'm thinking we can return an empty promise.

Reply:
`true` is success, while `false` means something went wrong. its true we don't use that information anywhere right now, but i still think it's generally useful (see #6586 for more thoughts on this pattern). i'm happy to add some docs to this function to clarify, do you think that is sufficient @iAmWillShepherd?

Reply:
That makes sense, but could we create a type for that so we get back `Success<T>` or `Failure` where `Failure` is an object that has some other info we may need to handle the failure?

As for this PR, adding docs is sufficient.

Reply:
absolutely, i would love that! i'm going to add a link to this comment in that issue